passage/src/main/java/id/passage/android/PassagePasskey.kt

```diff
@@ -0,0 +1,94 @@
+                    ?: AuthenticatorAttachment.platform
+            val webauthnStartRequest = RegisterWebAuthnStartRequest(identifier, authenticatorAttachment)
+            val webauthnStartResponse = registerAPI.registerWebauthnStart(Passage.appId, webauthnStartRequest)
+            // Use Create Credential challenge to prompt user to create a passkey
+            val createCredOptionsJson = PasskeyUtils.getCreateCredentialOptionsJson(webauthnStartResponse.handshake)
+            val createCredResponse = PasskeyUtils.createPasskey(createCredOptionsJson, activity)
+            // Complete registration and authenticate the user
+            val handshakeResponse = PasskeyUtils.getCreateCredentialHandshakeResponse(createCredResponse)
+            val webauthnFinishRequest =
+                RegisterWebAuthnFinishRequest(
+                    handshakeId = webauthnStartResponse.handshake.id,
+                    handshakeResponse = handshakeResponse,
+                    userId = webauthnStartResponse.user?.id ?: "",
+                )
+            val authResponse = registerAPI.registerWebauthnFinish(Passage.appId, webauthnFinishRequest)
+            // Handle and return auth result
+            // handleAuthResult(authResponse.authResult)
+            return authResponse.authResult
+        } catch (e: Exception) {
+            throw RegisterWithPasskeyException.convert(e)
+        }
+    }
+
+    /**
+     * Log in user with passkey
+     *
+     * Prompt the user to select a passkey to login with and authenticate the user.
+     * @param identifier valid email or E164 phone number
+     * @return PassageAuthResult
+     * @throws LoginWithPasskeyException
+     */
+    suspend fun login(identifier: String? = null): AuthResult {
+        try {
+            val loginAPI = LoginAPI(Passage.BASE_PATH, passageClient)
+            // Get Credential challenge from Passage
+            val webauthnStartRequest = LoginWebAuthnStartRequest(identifier)
+            val webauthnStartResponse = loginAPI.loginWebauthnStart(Passage.appId, webauthnStartRequest)
+            // Use Credential challenge to prompt user to login with a passkey
+            val credOptionsJson = PasskeyUtils.getCredentialOptionsJson(webauthnStartResponse.handshake)
+            val credResponse = PasskeyUtils.getPasskey(credOptionsJson, activity)
+            // Complete login and authenticate the user
+            val handshakeResponse = PasskeyUtils.getCredentialHandshakeResponse(credResponse)
+            val webauthnFinishRequest =
+                LoginWebAuthnFinishRequest(
+                    handshakeId = webauthnStartResponse.handshake.id,
+                    handshakeResponse = handshakeResponse,
+                    userId = webauthnStartResponse.user?.id,
+                )
+            val authResponse = loginAPI.loginWebauthnFinish(Passage.appId, webauthnFinishRequest)
+            // handleAuthResult(authResponse.authResult)
```

Comment:
Shouldn't we be saving the tokens in the new way here?
```suggestion
            tokenStore.setTokens(authResponse.authResult)
```